packages/front-end/enterprise/components/Dashboards/DashboardEditor/DashboardEditorSidebar/MetricExplorerSettings.tsx

```diff
@@ -52,178 +70,373 @@ export default function MetricExplorerSettings({ block, setBlock }: Props) {
-        placeholder="Select value"
-        setValue={(v) =>
-          setBlock({
-            ...block,
-            visualizationType: v as "bigNumber" | "timeseries" | "histogram",
-          })
-        }
-      >
-        <SelectItem value="bigNumber">Big Number</SelectItem>
-        <SelectItem value="timeseries">Timeseries</SelectItem>
-        {metric?.metricType === "mean" && (
-          <SelectItem value="histogram">Histogram</SelectItem>
-        )}
-      </Select>
+        <Collapsible
+          trigger={
+            <div
+              style={{
+                paddingLeft: "12px",
+                paddingRight: "12px",
+                paddingTop: "12px",
+                paddingBottom: "12px",
+                borderBottom: "1px solid var(--gray-a3)",
+              }}
+            >
+              <Text style={{ color: "var(--color-text-mid)", fontWeight: 500 }}>
+                <Flex justify="between" align="center">
+                  <Flex align="center" gap="1">
+                    <PiSlidersHorizontal
+                      style={{
+                        color: "var(--violet-11)",
+                      }}
+                      size={20}
+                    />
+                    Filters
+                    <Badge
+                      label={
+                        (
+                          (block.analysisSettings.numeratorFilters?.length ||
+                            0) +
+                          (block.analysisSettings.denominatorFilters?.length ||
+                            0)
+                        ).toString() || "0"
+                      }
+                      color="violet"
+                      radius="full"
+                      variant="soft"
+                    />
+                  </Flex>
+                  <Flex align="center" gap="1">
+                    <Button
+                      variant="ghost"
+                      color="red"
+                      disabled={
+                        block.analysisSettings.numeratorFilters?.length === 0 &&
+                        block.analysisSettings.denominatorFilters?.length === 0
+                      }
+                      onClick={() => {
+                        setBlock({
+                          ...block,
+                          analysisSettings: {
+                            ...block.analysisSettings,
+                            numeratorFilters: [],
+                            denominatorFilters: [],
+                          },
+                        });
+                      }}
+                    >
+                      Clear
+                    </Button>
+                    <FaAngleRight className="chevron" />
+                  </Flex>
+                </Flex>
+              </Text>
+            </div>
+          }
+          transitionTime={100}
+        >
+          <Box p="4" height="fit-content">
+            <Flex direction="column" gap="4">
+              <MultiSelectField
```

Comment:
I know we discussed this offline, and I don't think there's a good way to do it, but is there a way to indicate these are "additional" metric filters? A tooltip is the worst way to do this: "Row Filters specified here are combined with any existing filters on the metric.".

Fine if you want to override to keep it cleaner, I don't think it's a big deal.